packages/README.md

```diff
@@ -0,0 +1,6 @@
+## Kibana-related Packages
```

Comment:
thoughts on s/packages/node modules on this line?  i'd like to make sure there's a clear distinction between kibana's built tar/zip/deb/rpm packages and kibana's node dependent node modules